sites/main-site/src/content/events/2024/hackathon-barcelona.mdx

```diff
@@ -161,6 +161,19 @@ You can move freely between projects throughout the event.
 
 </HackathonGroup>
 
+<HackathonGroup title="nf-core/seqinspector" color="#238636" leaders={{'FranBonath': 'Franziska Bonath'}}>
+    nf-core/seqinspector aims to become a pipeline, specific for quality control of sequence data. Input is either fastq files or a run folder, and output is one global MultiQC report and, if wished, MultiQC files of groups that are defined in the sample sheet.
```

Comment:
```suggestion
    nf-core/seqinspector aims to be a pipeline for initial quality control of sequencing data. Input is either FASTQ files or a run folder, and output is planned to be a global MultiQC report and, if wished, MultiQC files of groups that are defined in the sample sheet.
```
For illumina sequencing only?

The all caps FASTQ is personal preference, can iugnore ;)

Reply:
since you have a preference and I don't, I am more than happy to oblige :)